internal/gengapic/genrest.go

```diff
@@ -396,6 +393,8 @@ func (g *generator) generateQueryString(m *descriptor.MethodDescriptorProto) {
 	sort.Strings(fields)
 
 	p("params := url.Values{}")
+	p(`params.Add("$alt", "json;enum-encoding=int")`)
```

Comment:
Does the dollar sign have significance? We set this param for discovery clients too, but just as `alt`

Reply:
@vchudnov-g ?

I was following what was described in the the linked bug, I'm not 100% sure tbh.

Reply:
Ah it is documented as supporting both: https://cloud.google.com/apis/docs/system-parameters.

Reply:
Google system params accept both. It seems to me it's safer to include the dollar sign, as you've done.